### ISSUE TITLE
error: [Errno 32] Broken pipe 

### ISSUE THREAD

**Issue by bfagundez** [gevent/gevent#445]
Using Flask 0.10.1, gevent 1.0. This is returned from time to time for different clients.
Not sure what happens but at some point this seems to create open files and hangs the entire server.

```
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/gevent/pywsgi.py", line 508, in handle_one_response
    self.run_application()
  File "/usr/local/lib/python2.7/dist-packages/gevent/pywsgi.py", line 495, in run_application
    self.process_result()
  File "/usr/local/lib/python2.7/dist-packages/gevent/pywsgi.py", line 486, in process_result
    self.write(data)
  File "/usr/local/lib/python2.7/dist-packages/gevent/pywsgi.py", line 380, in write
    self._write_with_headers(data)
  File "/usr/local/lib/python2.7/dist-packages/gevent/pywsgi.py", line 400, in _write_with_headers
    self._sendall(towrite)
  File "/usr/local/lib/python2.7/dist-packages/gevent/pywsgi.py", line 355, in _sendall
    self.socket.sendall(data)
  File "/usr/local/lib/python2.7/dist-packages/gevent/socket.py", line 458, in sendall
    data_sent += self.send(_get_memory(data, data_sent), flags)
  File "/usr/local/lib/python2.7/dist-packages/gevent/socket.py", line 443, in send
    return sock.send(data, flags)
error: [Errno 32] Broken pipe
{'GATEWAY_INTERFACE': 'CGI/1.1',
 'HTTP_ACCEPT': 'text/css,*/*;q=0.1',
 'HTTP_ACCEPT_ENCODING': 'gzip,deflate,sdch',
 'HTTP_ACCEPT_LANGUAGE': 'es-419,es;q=0.8',
 'HTTP_CONNECTION': 'keep-alive',
 'HTTP_HOST': 'www.mysitehidden.com',
 'HTTP_REFERER': 'http://www.mysitehidden.com/?utm_source=tenfield&utm_medium=display&utm_campaign=300int',
 'HTTP_USER_AGENT': 'Mozilla/5.0 (Linux; Android 4.4.2; XT1058 Build/KXA20.16-1.26) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/35.0.1916.122 Mobile Safari/537.36',
 'PATH_INFO': '/static/gen/packed.css',
 'QUERY_STRING': '9216d96c',
 'REMOTE_ADDR': '186.105.178.137',
'REMOTE_PORT': '36642',
 'REQUEST_METHOD': 'GET',
 'SCRIPT_NAME': '',
 'SERVER_NAME': 'ip-172-31-35-51.us-west-2.compute.internal',
 'SERVER_PORT': '80',
 'SERVER_PROTOCOL': 'HTTP/1.1',
 'SERVER_SOFTWARE': 'gevent/1.0 Python/2.7',
 'werkzeug.request': None,
 'wsgi.errors': <open file '<stderr>', mode 'w' at 0x7fcf0d0fa1e0>,
 'wsgi.input': <gevent.pywsgi.Input object at 0x38f2550>,
 'wsgi.multiprocess': False,
 'wsgi.multithread': False,
 'wsgi.run_once': False,
 'wsgi.url_scheme': 'http',
 'wsgi.version': (1, 0)} failed with error
```

**Comment by payne92** [comment]
+1

**Comment by roganov** [comment]
Is there a workaroung if no patch is coming soon?

**Comment by vim-zz** [comment]
+1

**Comment by jamadden** [comment]
Merged #377.
